android/keystore_1.md

```diff
@@ -0,0 +1,54 @@
+# Android Keystore
+
+## Introduction on the keystores
+
+Keystores are files containing security certificates and are primarily used to
+sign Android APK files. Usually you'll manage a single keystore that is used for
+your app in production. There is also a debug keystore that is generated by the
+Android SDK. You can check its location:
+
+```shell
+ls $HOME/.android/debug.keystore
+```
+
+If the keystore is not existing, check that you have installed the Android SDK.
+
+Keystores can be **password protected**: when it's the case, you'll be prompted for
+the password when you want to use it to sign your app.
+
+## Create a release keystore
+
+```shell
+keytool -genkey -v -keystore my-release-key.keystore
+-alias alias_name -keyalg RSA -keysize 2048 -validity 10000
```

Comment:
Should we talk a little bit about what each parameter means? maybe not for validity which is self explanatory, but you could add a line about the my-release-key.keystore